### DISCUSSION TITLE
XDG Desktop Entry insomnia.desktop

### DISCUSSION THREAD

**Discussion by noraj** [Kong/insomnia#5452] [Ideas] [answered]
I wanted to PR about the XDG Desktop Entry `insomnia.desktop` to add keywords to it. But the project is huge and so use a lot of abstraction. AFAIK it looks like there is no pre-build `.desktop` file but that it is rather built dynamically.

I found the issue https://github.com/Kong/insomnia/issues/1394 that mention that for example the `Categories` field of teh desktop entry is extracted from the `category` node of the [package.json](https://github.com/Kong/insomnia/blob/b3754a07e3f6976d05dc991a73786740b1ed1a38/package.json#L105). But with the latest version all the metadata have disappeared from it.

I would like to submit this desktop entry but I don't know where to PR it.

`insomnia.desktop`

```ini
[Desktop Entry]
Name=Insomnia
Comment=Debug APIs like a human, not a robot
Exec=/opt/insomnia/insomnia %U
Terminal=false
Type=Application
Icon=insomnia
MimeType=x-scheme-handler/insomnia;
Categories=Development;
Keywords=GraphQL;REST;gRPC;SOAP;openAPI;GitOps;
```

**Comment by filfreire** [comment, marked as answer]
Hi @noraj 

The `.desktop` file is auto generated by [electron builder config](https://www.electron.build/configuration/linux).

You can find the current lines we have here: https://github.com/Kong/insomnia/blob/develop/packages/insomnia/electron-builder.config.js#L105-L109

Feel free to submit a pull request adding some more information for the `.desktop`
